localization/twist_estimator/pose2twist/src/README.md

```diff
@@ -0,0 +1,32 @@
+# pose2twist
+
+## Purpose
+
+This `pose2twist` calculates the velocity from the input pose history. In addition to the computed twist, this node outputs the linear-x and angular-z components as a float message to simplify debugging.
+
+The `twist.linear.x` is calculated as `sqrt(dx * dx + dy * dy + dz * dz) / dt`, and the values in the `y` and `z` field are zero.
+The `twist.angular` is calculated as `d_roll / dt`, `d_pitch / dt` and `d_yaw / dt` for each field.
+
+## Inputs / Outputs
+
+### Input
+
+| Name | Type                            | Description                                      |
+| ---- | ------------------------------- | ------------------------------------------------ |
+| pose | geometry_msgs::msg::PoseStamped | pose source to used for the velocity calculation |
```

Comment:
IMO: Might need to add a period to Description, or remove periods in the other table.
This rule is not unified in all READMEs, so not sure which would be better.

Reply:
Thank you. I added it.

Reply:
Not be fixed yet?

Reply:
Ohh, sorry. I'll create a new PR.

Reply:
Thanks!!